phpunit-integration-legacy.xml

```diff
@@ -16,7 +16,7 @@
         <env name="fixtureDir" value="Legacy" />
         <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
         <ini name="error_reporting" value="-1" />
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="disabled"/>
```

Comment:
Looks like accidental removal. It should stay.

Reply:
It duplicates an identical line (:17)